src/lib/components/breadcrumbs.svelte

```diff
@@ -103,6 +103,7 @@
         isLoadingProjects = true;
         // null on non-org/project path like `onboarding`.
         loadedProjects = (await projects) ?? loadedProjects;
+        // Fallback logic removed; handled in loader
```

Comment:
the fallback needs to be added here because these projects are loaded via `await projects` which is supplied by this reactive variable here - https://github.com/appwrite/console/blob/6de1ec9f78cb10d3d2fa573ae16cb99c1c487eb0/src/routes/(console)/%2Blayout.svelte#L323C5-L333C14